angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -21,8 +21,8 @@ namespace :db_connector do
 
   def nyc_geocode(house_number, street, boro)
 
-    app_key = ENV['NY_GEOCLIENT_APP_ID']
-    app_secret = ENV['NY_GEOCLIENT_APP_KEY']
+    app_key = ENV['NY_GEOCLIENT_APP_KEY']
+    app_secret = ENV['NY_GEOCLIENT_APP_SECRET']
```

Comment:
This is wrong - .env.example currently uses ID and KEY, matching the terminology used on the Geoclient sign-up website. I switched the naming over from KEY and SECRET a few months ago in #87.